src/patternfly/upgrade-examples/LoginUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.login-pf` | `<div>` | `.pf-c-login` | `<div>` | Creates a login containing content. |
+| `.login-pf-heading` | `<div>` | `.pf-c-login__header` | `.pf-c-login` | Creates the header of a login. |    
+| `.login-pf-body` | `<div>` | `.pf-c-login__body` | `.pf-c-login` | Creates the body of a login. Required. | 
```

Comment:
maybe this should be `.card-pf` for pf3? I'm not seeing `.login-pf-body`. Also not seeing `.pf-c-login__body` on pf4. Looks like that should be `.pf-c-login__main` instead, and it applies to `<main>`, not `.pf-c-login`

Reply:
`.card-pf` looks like a class that only exists in PF3, I don't see the equivalent in PF4 for the login component. Instead of writing `.card-pf`, I narrowed down the elements with a child combinator (`>` ) method usage table. It's the only way to understand PF4's `.pf-c-login__main`. What do you think @mcoker?

Reply:
`.card-pf` seems to correspond to `.pf-c-login__main`